src/shared/extensionUtilities.ts

```diff
@@ -22,13 +22,15 @@ const localize = nls.loadMessageBundle()
 const vscodeAppname = 'Visual Studio Code'
 const cloud9Appname = 'AWS Cloud9'
 const cloud9CnAppname = 'Amazon Cloud9'
+const sageMakerAppname = 'Code - OSS'
```

Comment:
can this be `AWS SageMaker Code Editor` or something with "AWS SageMaker" in the name? 

`vscode.env.appName` is what is reported in telemetry, so a generic name like `Code - OSS` will be very obscure.

alternatively, we could special-case this [here](https://github.com/aws/aws-toolkit-vscode/blob/44a3bde971507267fd5dfb0832106fa40c6dafe4/src/shared/telemetry/telemetryClient.ts#L94) (and [here](https://github.com/aws/aws-toolkit-vscode/blob/44a3bde971507267fd5dfb0832106fa40c6dafe4/src/shared/telemetry/telemetryClient.ts#L120)), but even so, depending on a appName as generic as "Code - OSS" could run into false-positives in the wild.

Reply:
revised to `SageMaker Code Editor`